### PR TITLE
ROX-25048: add ui e2e test

### PR DESCRIPTION
**Pull request by stehessel**
### Description

Add a ui-e2e test for the S3 compatible form added in #11847.

### User-facing documentation

(*must be* 2 items and both *must be* checked)

- [ ] CHANGELOG is updated
- [x] CHANGELOG update is not needed
- [ ] Documentation PR is created and linked above
- [x] Documentation is not needed

### Testing

- [ ] inspected CI results

#### Automated testing

(*must be* at least 1 item and all items *must be* checked)

- [ ] added unit tests
- [ ] added e2e tests
- [ ] added regression tests
- [ ] added compatibility tests
- [ ] modified existing tests
- [x] contributed **no automated tests**
  

#### How I validated my change

See CI

### CODE REVIEW

**Review by lvalerom — COMMENTED** [review]
Maybe I'm missing something here but is the `externalBackups.json` used at all? I'm not very versed in the UI development but I tried to find references to the file and I couldn't find any 🤔

File: ui/apps/platform/cypress/fixtures/integrations/externalBackups.json [hunk 1]
```diff
@@ -23,6 +23,29 @@
...
+            "s3compatible": {
+                "bucket": "test",
+                "accessKeyId": "******",
+                "secretAccessKey": "******",
+                "region": "test",
+                "objectPrefix": "test",
+                "endpoint": "test",
+                "urlStyle": "S3_URL_STYLE_PATH"
+            }
+        },
```
Comment (sachaudh) [inline ui/apps/platform/cypress/fixtures/integrations/externalBackups.json:48]: @stehessel Considering @lvalerom  earlier comment, does it make sense to modify this file? I searched for references to this file and didn't find any. We can try undoing the changes to this file and see if everything still works as intended.

**Review by sachaudh — APPROVED** [review]
Looks good to me. Let me know if the changes in externalBackups.json ends up not being necessary.

File: ui/apps/platform/cypress/fixtures/integrations/externalBackups.json (hunk 1 above)
Comment (stehessel) [reply ui/apps/platform/cypress/fixtures/integrations/externalBackups.json:48]: I'll merge this now to keep the file consistent. But I'll create another PR to remove the files together with the other presumably unused fixtures.

**Pull request merged**